redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/TreehouseView.kt

```diff
@@ -24,22 +24,22 @@ import kotlinx.coroutines.flow.StateFlow
 import kotlinx.serialization.json.Json
 
 @ObjCName("TreehouseView", exact = true)
-public interface TreehouseView {
-  public val children: Widget.Children<*>
+public interface TreehouseView<W : Any> {
+  public val children: Widget.Children<W>
```

Comment:
Having `children` parametrised is technically the only requirement for having `TreehouseView` extend from `RedwoodView`. The other parametrisations within the file are purely in the hope of improving type safety.